src/workloads/query/PercentilesAgg_Distributions.yml

```diff
@@ -0,0 +1,226 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  Run tests for $percentile accumulator over various data distributions. We create a new collection
+  for each distribution as putting the fields into the same document might be impacted by how much
+  bson needs to be parsed to access the field but we want to make sure we compare apples to apples.
+
+Keywords:
+- group
+- percentile
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - standalone-all-feature-flags
```

Comment:
We put down replica set just because we want to run the feature in both SBE and classic engine to compare the runtime when this feature eventually gets added to SBE, and we did not see an option in the form to have both options in a standalone. However, we don't need to run a replica set (the feature doesn't use any replication features). So is there a pair of build variants using standalone (one for SBE and one for classic)?

Reply:
I see, thanks for the feedback I'll let the team know to update the form. These two variants should fit the bill for your use case:

- linux-standalone-classic-query-engine.2022-11

- linux-standalone-sbe.2022-11

Reply:
Thank you! Those would work. Should we update our response to the form when these fields are added?

Reply:
You should be okay to leave your submission as is.